bigbluebutton-html5/imports/api/external-videos/server/handlers/startExternalVideo.js

```diff
@@ -1,23 +1,13 @@
 import { check } from 'meteor/check';
-import Logger from '/imports/startup/server/logger';
-import Users from '/imports/api/users';
-import Meetings from '/imports/api/meetings';
+import startExternalVideo from '../modifiers/startExternalVideo';
 
 export default function handleStartExternalVideo({ header, body }, meetingId) {
-  const { userId } = header;
+  check(header, Object);
   check(body, Object);
   check(meetingId, String);
-  check(userId, String);
 
-  const externalVideoUrl = body.externalVideoUrl;
-  const user = Users.findOne({ meetingId: meetingId, userId: userId })
+  const { userId } = header;
```

Comment:
We don't check if `userId` and `externalVideoUrl` are strings neither here nor in akka-apps, we just start using them